helm/loki/values.yaml

```diff
@@ -60,6 +60,7 @@ serviceAccount:
   automountServiceAccountToken: true
 
 loki:
+  genMultiTenantNginx: false
```

Comment:
What is that property?

Reply:
It's the "gateway.nginxConfig.genMultiTenant", but at the wrong place :grimacing: 
Fixed.

Reply:
Nice